Source/utils/attributes.h

```diff
@@ -31,3 +31,14 @@
 #else
 #define DVL_ATTRIBUTE_HOT
 #endif
+
+// Any global data used by tests must be marked with `DVL_API_FOR_TEST`.

```

Comment:
are we running in to issues that makes this necessary?

Reply:
Now that we link libdevilutionx to tests as a shared library, all global data must be marked this way for MSVC to export it.
For functions and classes we set the `WINDOWS_EXPORT_ALL_SYMBOLS` property but it doesn't work for global variables.

The alternative would be to link libdevilutionx statically into each test binary but that is much slower and would take lots of disk space.

Reply:
> Now that we link libdevilutionx to each test binary as a shared library

Well this is the part I don't understand why is needed :)

Reply:
> The alternative would be to link libdevilutionx statically into each test binary but that is much slower and would take lots of disk space.

^ basically this